README.md

```diff
@@ -68,26 +75,29 @@ Testing is currently only supported in regions with the 2060 but it should show
 | :memo:        | All commands should be executed inside of cmd.exe |
 |---------------|:------------------------|
 ```Batchfile
-nvidia-clerk-windows.exe -region=USA -test -model=2060
+nvidia-clerk-windows.exe -region=USA -model=2060
 ```
 
 ### Mac OSX
 | :memo:        | All commands should be executed inside of Terminal.app |
 |---------------|:------------------------|
 ```shell
-./nvidia-clerk-darwin -region=USA -test -model=2060
+./nvidia-clerk-darwin -region=USA -model=2060
 ```
 
 ### Linux
 | :memo:        | All commands should be executed inside of Shell |
 |---------------|:------------------------|
 ```shell
-./nvidia-clerk-linux -region=USA -test -model=2060
+./nvidia-clerk-linux -region=USA -model=2060
 ```
 
 
 # Advanced Usage
 
+## Regional Product Listing
```

Comment:
Lets just remove this for now I'll add it again later.

Reply:
ok